### ISSUE TITLE
Animate faq caret upon click

### ISSUE THREAD

**Issue by joshuagad** [gcfyouthlive/camplive#7]
In the faqs, upon clicking the link, the caret should rotate 45 degrees. When the faq is clicked again (closed), the care will rotate back to initial position.

**Comment by montanohanz** [comment]
Didnt use css anims, rather used icon replaces using jQ
Commit: 8aaf5c80649c91c2bcc3154b0e9fb1d57ef41c1d

**Comment by joshuagad** [comment]
Hmm. Check this out and see if you can try this: https://stackoverflow.com/questions/26173376/rotate-font-awesome-icon-on-click

By rotating instead of changing, I think we can see a smoother transition of the text moving to the right? So it haa the same effect as the collapsed div opening.

**Comment by montanohanz** [comment]
Ohh tried this na kanina kuya but it didnt work? (or i just really suck at js HAHA) ill try combining with the current script

**Comment by joshuagad** [comment]
Check out my latest commit on feature/faqs. I used jquery to attach an event to the bootstrap collapse upon show (check bootstrap docs). The event modifies the caret's css to rotate 90 degrees. I also created an "opposite" event for hide. That way, double-clicking won't break the caret. It is tied to the collapse, mot the click.

Then I set css transition to 0.3s. Check it out. :) 

Lesson learned: css animations only work when changing css. It doesn't work when toggling a class only.
